datadog-checks-base/datadog_checks/stubs/aggregator.py

```diff
@@ -77,6 +77,28 @@ def reset(self):
         self._asserted = set()
         self._service_checks = defaultdict(list)
 
+    def all_metrics_asserted(self):
```

Comment:
Not sure this should print anything, or if it prints it should have another name. If we decide to print something, we should agree on a specific format to avoid standard proliferation.

Reply:
That's a good point

Reply:
I removed the print statements we can sort this out in another pr